kube-runtime/src/scheduler.rs

```diff
@@ -1,4 +1,4 @@
-//! Delays and deduplicates [`Stream`](futures::stream::Stream) items
+//! Delays and deduplicates [`Stream`] items
```

Comment:
has rustdoc gotten smarter lately? this stuff used to give broken links when running `just doc`

Reply:
Ah, damn. You're right.. clippy lead me astray. Will revert.

Reply:
Nevermind, this actually seems to have come from the `--no-deps` flag given by `just doc`. Removing that generates the links correctly.

Reply:
hopefully this is how docs.rs builds docs. i tried to find some info on this, but not much on [their docs](https://docs.rs/about/builds). it is possible to [inject rustdoc args](https://docs.rs/about/metadata) though, so should probably be fine

Reply:
Looks like the code that controls the arguments is https://github.com/rust-lang/docs.rs/blob/5ec4326126b2f244442f55f0183ca7523611f0df/crates/metadata/lib.rs#L253-L300. Building it as `RUSTC_BOOTSTRAP=1 cargo rustdoc --lib -Zrustdoc-map --all-features -- --cfg docsrs` does seem to work, and links to the corresponding docs.rs entries.